app/views/support_interface/providers/sites.html.erb

```diff
@@ -9,9 +9,15 @@
   </thead>
 
   <tbody class='govuk-table__body'>
-    <% @provider.sites.each do |site| %>
+    <% @provider.sites.includes(:course_options).each do |site| %>
       <tr class='govuk-table__row'>
-        <td class='govuk-table__cell'><%= site.name_and_code %></td>
+        <td class='govuk-table__cell'>
+          <%= site.name_and_code %>
+
+          <% if site.deleted_from_find? %>
+            <%= render TagComponent.new(text: 'Deleted in Find', type: 'red') %>
+          <% end %>
```

Comment:
This is just wrong, the site isn't actually deleted AFAWK, just that there are courses that used to be offered at the site, that no longer are.